python/runtime/cudaq/algorithms/py_sample.cpp

```diff
@@ -95,14 +104,12 @@ void bindSample(py::module &mod) {
 
   py::class_<async_sample_result>(
       mod, "AsyncSampleResult",
-      "A data-type containing the results of a call to :func:`sample_async`. "
-      "The `AsyncSampleResult` contains a future, whose "
-      ":class:`SampleResult` "
-      "may be returned via an invocation of the `get` method. This "
-      "kicks "
-      "off a wait on the current thread until the results are available.\n"
-      "See `future <https://en.cppreference.com/w/cpp/thread/future>`_ "
-      "for more information on this programming pattern.\n")
+      R"#(A data-type containing the results of a call to :func:`sample_async`. 
+      The `AsyncSampleResult` models a future-like type, whose 
+      :class:`SampleResult` may be returned via an invocation of the `get` method. This 
+      kicks off a wait on the current thread until the results are available.
+      See `future <https://en.cppreference.com/w/cpp/thread/future>`_ 
+      for more information on this programming pattern.)#")
```

Comment:
Do you want this indentation as part of the literal string?

Reply:
Nope